src/Sulu/Bundle/WebsiteBundle/DependencyInjection/Configuration.php

```diff
@@ -56,6 +56,13 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('sitemap')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->scalarNode('dump_dir')->defaultValue('%sulu.cache_dir%/sitemaps')->end()
+                    ->scalarNode('default_host')->defaultNull()->end()
```

Comment:
we also thought about that but currently this is the only place and if there will be some other places it is not said that it should not be defined default then?